ydb/docs/en/core/integrations/grafana.md

```diff
@@ -0,0 +1,136 @@
+
+### Configuration via UI
+
+Once the plugin is installed on your Grafana instance, follow [these instructions](https://grafana.com/docs/grafana/latest/datasources/add-a-data-source/) to add a new {{ ydb-short-name }} data source, and enter configuration options.
+
+### Configuration with provisioning system
+
+Alternatively, it is possible to configure data sources using configuration files with Grafana’s provisioning system. To read about how it works, including all the settings that you can set for this data source, refer to [Provisioning Grafana data sources](https://grafana.com/docs/grafana/latest/administration/provisioning/#data-sources) documentation.
+
+### Authentication
+
+The Grafana plugin supports various authentication [authentication methods](../reference/ydb-sdk/auth.md).
+
+Below is an example config for authenticating a {{ ydb-short-name }} data source using username and password:
+
+```yaml
+apiVersion: 1
+datasources:
+  - name: YDB
+    type: ydbtech-ydb-datasource
+    jsonData:
+      authKind: "UserPassword"
+      endpoint: 'grpcs://<hostname>:2135'
+      dbLocation: '/eu-central1/b1g/db-location'
+      user: '<username>'
+    secureJsonData:
+      password: '<userpassword>'
+      certificate: 'content of *.pem file'
+```
+
+Here are fields that are supported in connection configuration:
+
+| Name  | Description         |         Type          |
+| :---- | :------------------ | :-------------------: |
+| authKind | Authentication type |       `"Anonymous" \| "ServiceAccountKey" \| "AccessToken" \| "UserPassword" \| "MetaData"`        |
+| endpoint | Database endpoint  | `string` |
+| dbLocation | Database location  | `string` |
+| user | User name  | `string` |
+| serviceAccAuthAccessKey | Service account access key  | `string` (secured) |
+| accessToken | OAuth access token  | `string` (secured) |
+| password | User password  | `string` (secured) |
+| certificate | Content of *.pem file  | `string` (secured) |
+
+## Building queries
+
+{{ ydb-short-name }} is queried with a SQL dialect named [YQL](../yql/reference/index.md).
+Queries can contain macros which simplify syntax and allow for dynamic parts. It may be two kind of macro - [grafana-level](#macros) and ydb-level. Plugin will parse query text and, before sending it to YDB, substitute variables and grafana-level macroses with particular values. After that ydb-level macroses will be treated by ydb. 
+The query editor allows you to get data in different representation: time series, table or logs.
+
+### Time series (../_assets/grafana/time-series.png)
+
+Time series visualization options are selectable if the query returns at least one field with `Date`, `Datetime`, or `Timestamp` type type and at least one field with `int`, `uint`, `double` or `float` type. Then you can select time series visualization options. Grafana interprets timestamp rows without explicit time zone as UTC. Any other column is treated as a value column.
```

Comment:
What does a row with/without explicit timezone mean? In YDB there are separate TzDate/TzDatetime/TzTimestamp data types for timezone handling, from the previous sentence it sounds like they are not supported.

Reply:
TzDate/TzDatetime/TzTimestamp are not supported.
`What does a row with/without explicit timezone mean?` - I don't understand your question. If no explicit timezone column is given, you won't get Time series view.